scripts/promote-charms.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash -e
+#
+# Anbox - The Android in a Box runtime environment
+# Copyright 2024 Canonical Ltd.  All rights reserved.
+#
+
+from=
+to=
+charms=("ams" "ams-lxd" "coturn" "anbox-stream-gateway" "anbox-stream-agent" "anbox-cloud-dashboard" "ams-node-controller" "aar" )
```

Comment:
And we want the `nats` charm here?
BTW, for the nats, is the plan going to have dedicated channels rather than a single latest one in the future?

Reply:
Updated the script thanks. NATS charm only works on the latest afaik, however if we wanted we could look into putting the versions according to NATS version.